src/specification/models/curratedLocality/index.json

```diff
@@ -0,0 +1,71 @@
+{
+  "additionalProperties": false,
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Name of the locality"
+    },
+    "versionId": {
+      "type": "string",
+      "description": "Id of the specific version"
+    },
+    "createdAt": {
+      "type": "string",
+      "description": "Timestamp corresponding to when created"
+    },
+    "createdBy": {
+      "type": "string",
+      "description": "Some kind of reference to the individual that created the locality"
+    },
+    "updatedBy": {
+      "type": "string",
+      "description": "Some kind of reference to the individual that updated/created the specific version"
+    },
+    "updatedAt": {
+      "type": "string",
+      "description": "Timestamp corresponding to when updated. Could be called versionCreatedAt"
+    },
+    "id": {
+      "type": "string",
+      "description": "Unique identifier that is consistent over time"
+    },
+    "description": {
+      "type": "string",
+      "description": "Description of the locality"
+    },
+    "relations": {
+      "type": "array",
+      "description": "A relation to other curratedLocalities",
+      "items": { "$ref": "__ROOT__curratedLocalityRelation" }
+    },
+    "type": {
+      "type": "string",
+      "description": "Type of locality",
+      "enum": [
+        "continent",
+        "country",
+        "district",
+        "geodeticDatum",
+        "locality",
+        "province",
+        "rubin"
```

Comment:
sort the enums?

so a locality can have type `locality`? _might_ become hard to distinguish when talking about localities ("do you mean any type of locality or a locality of type locality") :-)